### PR TITLE
chore: merge released

### PR DESCRIPTION
**Pull request by guanbinrui**

### CODE REVIEW

**Comment by Jack-Works** [comment]
what is this PR 🤔

**Comment by septs** [comment]
image

**Comment by guanbinrui** [comment]
> what is this PR 🤔

Too many conflicts so I decided to open a PR for reviewing.

File: packages/maskbook/src/extension/options-page/DashboardDialogs/Wallet.tsx [hunk 1]
```diff
@@ -377,6 +440,134 @@ export function DashboardWalletAddERC20TokenDialog(props: WrappedDialogProps<Wal
...
+    const { t } = useI18N()
+    const { wallet, tokenIdsLoaded } = props.ComponentProps!
+    const [tokenId, setTokenId] = useState('')
+    const [address, setAddress] = useState('')
+    const account = useAccount()
+    const chainId = useChainId()
+
+    const onSubmit = useSnackbarCallback(
+        async () => {
+            const contract = createContract<ERC721>(account, address, ERC721ABI as AbiItem[])
```
[the remark this answers is not in the document]
Comment (guanbinrui) [reply packages/maskbook/src/extension/options-page/DashboardDialogs/Wallet.tsx:463]: Resolved by #2731.

**Comment by Jack-Works** [comment]
> Too many conflicts when I merge released back to the master branch, so I decided to open a PR for review.

That's why I'm not encouraging you add new features in the patch branch

**Pull request merged**